extensions/wikia/SpecialDiscussionsLog/SpecialDiscussionsLogController.class.php

```diff
@@ -0,0 +1,230 @@
+<?php
+
+/**
+ * Discussion user log page
+ */
+class SpecialDiscussionsLogController extends WikiaSpecialPageController {
+	// @todo
+	//1. Pagination
+	//2. Add date range input field
+
+	const KIBANA_BASEURL = 'http://query.es.service.sjc.consul:9200';
```

Comment:
One of the AC is to "Remove any hardcoded values", I think this counts as hard-coded, yea? ;)

We actually have a class to resolve these urls: [ConsulUrlProvider.java](https://github.com/Wikia/app/blob/dev/lib/Wikia/src/Service/Gateway/ConsulUrlProvider.php). It takes a service tag ('query') and optional dc ('sjc'). So using it would look something like this:

```PHP
global $wgConsulUrl;
$urlProvider = new  Wikia\Service\Gateway\ConsulUrlProvider( $wgConsulUrl, 'query', 'sjc' );
$esUrl = $urlProvider->getUrl( 'es' );
```